CONTRIBUTING.md

```diff
@@ -84,10 +84,10 @@ We use the following [labels](https://github.com/plotly/plotly.js/labels) to tra
   [`package-lock.json`](https://docs.npmjs.com/files/package-lock.json) file is
   used and updated correctly.
 
-#### Step 1: Clone the plotly.js repo and install its dependencies
+#### Step 1: Clone the plotly.js repo, download it and install dependencies
 
 ```bash
-git clone https://github.com/plotly/plotly.js.git
+git clone --depth 1 git@github.com:<your-GitHub-username>/plotly.js.git
```

Comment:
Is `--depth 1` important enough to warrant including here? I could imagine it occasionally being dangerous, like if the user wanted to pick up an old branch and complete the work.

Also, in https://github.com/plotly/plotly.js/pull/5780 we talk about `origin` and `upstream`, do we want to discuss how to set that up?

Reply:
> Is `--depth 1` important enough to warrant including here? I could imagine it occasionally being dangerous, like if the user wanted to pick up an old branch and complete the work.
> 

I thought it will make it easier for the people to contribute, instead of pulling down the whole history of plotly.js.
Curious to know what @nicolaskruchten thinks here?

Reply:
> Also, in #5780 we talk about `origin` and `upstream`, do we want to discuss how to set that up?

![fetch-upstream](https://user-images.githubusercontent.com/33888540/124777961-94910400-df0e-11eb-9aa2-1cd747167bcc.png)
**It appears GitHub recently added a new button which made the process of fetching upstream way easier.**